backend/infrastructure/api/endpoints/client.py

```diff
@@ -0,0 +1,71 @@
+from fastapi import APIRouter, HTTPException, Request
+from pydantic import BaseModel
+from infrastructure.mongo.client_handler import register_client, get_client
+
+class Client(BaseModel):
+    email: str
```

Comment:
client its part of domain infrastructure shoould be submissive to the bussiness domain not in the other way

Reply:
(Part of business rule)